jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -602,10 +613,12 @@ public InternalCRIUSupport setWorkDir(Path workDir) {
 		}
 		String dir = workDir.toAbsolutePath().toString();
 
+		/*[IF JAVA_SPEC_VERSION < 24]*/
```

Comment:
It seems 
https://github.com/eclipse-openj9/openj9/blob/48e4702cf4dc15cb1b8e8f990695eb00be034014/jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java#L38-L40
should be moved to this file and changed to 
```
/*[IF (17 <= JAVA_SPEC_VERSION) & (JAVA_SPEC_VERSION < 24)]*/
@SuppressWarnings({ "deprecation", "removal" })
/*[ENDIF] (17 <= JAVA_SPEC_VERSION) & (JAVA_SPEC_VERSION < 24) */
```
because the deprecated `System.getSecurityManager()` usages are being removed.

Reply:
Thanks, I have made this change.

Reply:
I probably should have made it explicit, the `@SuppressWarnings({ "deprecation", "removal" })` need to be moved from `CRIUSupport.java` to `InternalCRIUSupport.java`.
The security manager API usages are in `InternalCRIUSupport.java` but not in `CRIUSupport.java` which doesn't need this annotation. I think this was due to earlier refactoring, please help fix it.

Reply:
I see what you mean now. I've moved it to `InternalCRIUSupport.java`.